### PR TITLE
CORE-1481 Add Popular Featured apps row to dashboard

### PR DESCRIPTION
**Pull request by ash3rz**
NOTE: The [terrain](https://github.com/cyverse-de/terrain/pull/226) and [dashboard-aggregator](https://github.com/cyverse-de/dashboard-aggregator/pull/10) PRs will need to be merged prior to this one.

image

### CODE REVIEW

File: public/static/locales/en/dashboard.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "origination": "{{origination}} {{user}} on {{date}}",
     "outputAction": "Show Output Files",
     "outputFilesAria": "go to output files",
+    "popularFeaturedApps": "Popular Featured Apps",
```
Comment (sri2k1us) [inline public/static/locales/en/dashboard.json:38]: Any reason to call it `Popular Featured Apps` instead of just `Featured Apps` ?

**Review by sri2k1us — APPROVED** [review]
:shipit:

File: public/static/locales/en/dashboard.json (hunk 1 above)
Comment (sri2k1us) [reply public/static/locales/en/dashboard.json:38]: Also I guess for logged out view we can replaced `Public Apps` with `Featured Apps`

**Review by psarando — APPROVED** [review]
LGTM 👍

**Comment by ash3rz** [comment]
Thanks for the reviews!

**Pull request merged**